test_env/scenario-net-down.sh

```diff
@@ -21,16 +21,16 @@ export HIDDEN_CONTAINER_ID=$(docker ps | grep hidden | awk '{print $1}'); echo $
 export EXPOSED_CLIENT_IP=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $EXPOSED_CONTAINER_ID); echo $EXPOSED_CLIENT_IP
 export HIDDEN_CLIENT_IP=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $HIDDEN_CONTAINER_ID); echo $HIDDEN_CLIENT_IP
 
-export EXPOSED_NODE_ID=$(docker logs $EXPOSED_CONTAINER_ID 2>&1 | grep "NODE ID" | awk '{print $7}' | tail -1); echo $EXPOSED_NODE_ID
-export HIDDEN_NODE_ID=$(docker logs $HIDDEN_CONTAINER_ID 2>&1 | grep "NODE ID" | awk '{print $7}' | tail -1); echo $HIDDEN_NODE_ID
+export EXPOSED_NODE_ID=$(docker logs $EXPOSED_CONTAINER_ID 2>&1 | grep "NODE ID" | sed -E 's/^.*(0x[a-f0-9]{40}).*$/\1/g' | tail -1); echo $EXPOSED_NODE_ID
+export HIDDEN_NODE_ID=$(docker logs $HIDDEN_CONTAINER_ID 2>&1 | grep "NODE ID" | sed -E 's/^.*(0x[a-f0-9]{40}).*$/\1/g' | tail -1); echo $HIDDEN_NODE_ID
 
 export NETWORK_OF_HIDDEN_CLIENT=$(docker container inspect -f '{{range $net,$v := .NetworkSettings.Networks}}{{printf "%s" $net}}{{end}}' $HIDDEN_CONTAINER_ID); echo $NETWORK_OF_HIDDEN_CLIENT
 
 # Ping hidden client from exposed client
-curl -w "Ping: exposed client -> hidden node\n" -X GET http://$EXPOSED_CLIENT_IP:8081/ping/$HIDDEN_NODE_ID
+curl -w "\n" -X GET http://$EXPOSED_CLIENT_IP:8081/ping/unreliable/$HIDDEN_NODE_ID
```

Comment:
Ping `unreliable` makes client to not receive `ping` msg after it becomes available.
Was it meant to fix this scenario, or was it done only to clean up logs?

Reply:
It is just a helper script for manual use. I added it because the original URL was not working as it missed the transport type.